code-challenges.test.js

```diff
@@ -56,4 +168,31 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
-// b) Create the function that makes the test pass.
+        describe("firstV", () => {
+                  it("logs the index of the first vowel", () => {
+                          expect(firstV()).toEqual(1)
```

Comment:
I don't see an argument when you are invoking the function.  This could be throwing the .length error.